integration-app/src/main/java/uk/gov/crowncommercial/esourcing/integration/service/GovNotifyEmailService.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.crowncommercial.esourcing.integration.service;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.service.notify.NotificationClient;
+import uk.gov.service.notify.NotificationClientException;
+
+@Service
+public class GovNotifyEmailService implements EmailService {
+
+  private static final Logger LOG = LoggerFactory.getLogger(GovNotifyEmailService.class);
+
+  @Value("${ccs.esourcing.gov.notify.apikey}")
+  private String govNotifyApiKey;
+
+  @Value("${ccs.esourcing.gov.notify.techops.email}")
+  private String techopsEmail;
+
+  @Value("${ccs.esourcing.gov.notify.salesforce.template.id}")
+  private String salesforceTemplateId;
+
+  public void sendSalesforceUpdateFailureEmail(Map<String, String> map) {
```

Comment:
Better as `sendSalesforceUpdateFailureEmails` as it may send more than one?

Reply:
done